configuration.yaml

```diff
@@ -28,15 +28,16 @@ helm-repo-lookup:
     path: stable/cray-hms-trs-operator
   - chart: cray-hms-hmcollector
     path: stable/hms-hmcollector
-repo-image-lookup:
-  - repo: hms-shcd-parser
+github-repo-image-lookup:
+  - github-repo: hms-shcd-parser
     image: hms-shcd-parser
-  - repo: hms-trs-worker-http-v1
+  - github-repo: hms-trs-worker-http-v1
     image: hms-trs-worker-http-v1
 configuration:
   manifest-repo: csm
   targeted-csm-branches:
     - main
+    - release/1.2
```

Comment:
should be stable/1.2

Reply:
Do we want all 3 branches: 
```
- main
- release/1.2
- stable/1.2
```

Or just these 2?
```
- main
- release/1.2
```

Reply:
Just these
```yaml
- main
- release/1.2
```